resources/views/upload.blade.php

```diff
@@ -55,14 +55,18 @@ class="{{ $attributes->get('class') }}"
       pond.setOptions({
           allowMultiple: isMultiple,
           server: {
-              process: (fieldName, file, metadata, load, error, progress) => {
-                  @this.upload('{{ $wireModelAttribute }}', file, load, error, progress);
+              process: async (fieldName, file, metadata, load, error, progress) => {
+                  $dispatch('filepond-upload-started');
```

Comment:
Can you add the `$wireModelAttribute` as an extra value that gets sent with the event as well?

Reply:
Yes sure, is there any other useful information i should pass in the event ?